dashboardsSanitizer/README.md

```diff
@@ -0,0 +1,16 @@
+# Dashboard Sanitizer
```

Comment:
For how this might be used more broadly, what do you think about advocating for folks to `git clone ...` this repo and then run the `./gradlew dashboardsSanitzer:run ...` argument below?